.github/workflows/actions.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           ${{ runner.os }}-pip-
     - name: Install dependencies
       run: |
-        pip install tensorflow>=2.13.0
+        pip install tf-nightly-cpu==2.16.0.dev20231103  # Pin a working nightly until rc0.
         pip install "jax[cpu]"
         pip install torch>=2.0.1+cpu
         pip install torchvision>=0.15.1
```

Comment:
We should be able to remove the keras-core line below this. Keras core will never be used in Keras 3 is installed.

Reply:
done!